addons/player-thumb/userstyle.css

```diff
@@ -0,0 +1,6 @@
+[class*="stage_green-flag-overlay-wrapper"] {
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+  background-color: white;
+}
```

Comment:
I'm not sure if it's within the scope of the addon to remove the tint from the green flag overlay.